components/web/apps/org.wso2.carbon.apimgt.publisher/src/main/package.json

```diff
@@ -28,7 +28,7 @@
     "react-router": "^4.1.1",
     "react-router-dom": "^4.1.1",
     "react-tap-event-plugin": "^2.0.1",
-    "swagger-client": "^3.0.10",
+    "swagger-client": "2.1.27",
```

Comment:
Version downgraded to previously used version due to the missing [feature(s)](https://github.com/swagger-api/swagger-js/issues/1045) in the new release until they get implemented.